bubblesub/api/gui.py

```diff
@@ -52,6 +52,21 @@ def set_main_window(self, main_window: QtWidgets.QWidget) -> None:
         """
         self._main_window = main_window
 
+    def is_widget_visible(self, widget_name: str) -> bool:
+        """Say if a widget is visible.
+
+        :param widget_name: name of the widget to look for
+        :return: true if the widget is visible, false otherwise
+        """
+        widget = self._main_window.findChild(
+            QtWidgets.QWidget, str(widget_name)
+        )
+
+        if widget is None:
+            self._api.log.error("{} cannot be found".format(widget_name))
```

Comment:
If that's the case and widget is `None`, it will cause an ugly exception at runtime.
Raise a custom exception here and catch it in the commands that use this method. Log the error there, in the commands.

Reply:
Sounds good